src/frontend/src/lib/components/views/AddAccessMethodDialog.svelte

```diff
@@ -0,0 +1,143 @@
+<script lang="ts">
+  import Dialog from "$lib/components/ui/Dialog.svelte";
+  import Button from "$lib/components/ui/Button.svelte";
+  import GoogleIcon from "$lib/components/icons/GoogleIcon.svelte";
+  import identityInfo from "$lib/stores/identity-info.state.svelte";
+  import { handleError } from "../utils/error";
+  import { authenticatedStore } from "$lib/stores/authentication.store";
+  import { StartAddPasskeyFlow } from "$lib/flows/startAddPasskeyFlow.svelte";
+  import QrCreator from "qr-creator";
+  import Input from "../ui/Input.svelte";
+  import { isDarkMode } from "$lib/state/UI/isDarkMode";
+  import { CROSS_DEVICE_PASSKEYS } from "$lib/state/featureFlags";
+
+  const { onClose } = $props();
+
+  const startAddPasskeyFlow = new StartAddPasskeyFlow(
+    $authenticatedStore.identityNumber,
+  );
+
+  let inputRef = $state<HTMLInputElement>();
+  let qrCodeRef = $state<HTMLDivElement>();
+
+  let startAddingPasskeyDialog = $state(false);
+  let url = $derived(
+    `${window.location.origin}/flow/?user=${$authenticatedStore.identityNumber}`,
+  );
+
+  $effect(() => {
+    if (qrCodeRef) {
+      // Remove existing children when dark mode settings change
```

Comment:
What does it mean "when dark mode settings change"? I don't see any code managing that.